Containers/mastercontainer/Dockerfile

```diff
@@ -34,6 +34,7 @@ RUN set -ex; \
         bash \
         apache2 \
         apache2-proxy \
+        apache2-ssl \
```

Comment:
This will a file called `/etc/apache2/conf.d/ssl.conf` or something like this, which contains "Listen 443" and a virtual host on port 443, we should remove the virtualhost and the listen line

Reply:
removed the ssl.conf